app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -49,6 +49,62 @@ class PatientRepository @Inject constructor(
 
   private var ongoingNewPatientEntry: OngoingNewPatientEntry = OngoingNewPatientEntry()
 
+  fun search(name: String): Observable<List<PatientSearchResult>> {
+    return configProvider.flatMapObservable { (isFuzzySearchV2Enabled) ->
```

Comment:
I'm slowly growing afraid of using destructuring because they're index based

Reply:
Good point. I'll remove the destructuring here. We should probably limit destructuring to things like `Pair`, `Triple` and `Map.Entry` going forward.